libs/s3/s3_blackbox_test_lib.py

```diff
@@ -65,13 +65,14 @@ def configure_jclient_cloud(
         dir_list = os.listdir(source)
         if "jcloudclient.jar" not in dir_list or "jclient.jar" not in dir_list:
             temp_dir = "/mnt/jjarfiles"
-            os.mkdir(temp_dir)
+            if not os.path.exists(temp_dir):
+                os.mkdir(temp_dir)
             mount_cmd = f"mount.nfs -v {nfs_path} {temp_dir}"
             umount_cmd = f"umount -v {temp_dir}"
             run_local_cmd(mount_cmd)
-            run_local_cmd(f"yes | cp -rf {temp_dir}*.jar {source}")
+            run_local_cmd(f"yes | cp -rf {temp_dir}/*.jar {source}")
```

Comment:
Better to use os package for local operations instead of using run_local_cmd.

Reply:
Since **run_local_cmd** is handling error and exception, can we use it here ? also, I see many uses of run_local_cmd.

Reply:
Agree if those are for Linux specific commands, for rest I would recommend os/shutil package.

Reply:
Using os or shutils for all jar files is making code lines to increase, keeping the original code as discussed